questions/20-medium-promise-all/template.ts

```diff
@@ -1 +1 @@
-declare function PromiseAll(promises: any[]): any
+declare function PromiseAll(values: ReadonlyArray<any>): Promise<Array<any>>
```

Comment:
```suggestion
declare function PromiseAll(values: ReadonlyArray<any>): any
```

Would this better to leave it as `any`?